src/pb_stub.cc

```diff
@@ -79,9 +79,10 @@ Stub::Instantiate(
     const std::string& shm_region_name, const std::string& model_path,
     const std::string& model_version, const std::string& triton_install_path,
     bi::managed_external_buffer::handle_t ipc_control_handle,
-    const std::string& name)
+    const std::string& name, const std::string& plugin_model)
```

Comment:
nit: `plugin_model` -> `plugin_model_path` for consistency with the function signature.

Reply:
This is actually the name of the platform_model(previously plugin_model)

Reply:
I see. Then I think we would need to change the `platform_model_path` to `platform_model` here https://github.com/triton-inference-server/python_backend/pull/255/files#diff-b5637c6a5b8dd3bddd8aa5c4b9f2ac30895e75fd678cdcf66b50c9b0d607e94eR188.